File: src/main/java/it/contrader/dto/RecensioneDTO.java
package it.contrader.dto;

import it.contrader.model.Prenotazione;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RecensioneDTO {

	private long id;
	private int voto;
	private String testo;
	private Prenotazione prenotazione;

}

File: src/dto/recensionedto.ts
import { PrenotazioneDTO } from './prenotazionedto';
import {UserDTO} from './userdto';


export class RecensioneDTO{
    id: number;
    voto: number;
    testo: String;
    prenotazioneId: number;
    prenotazione: PrenotazioneDTO;
}